client/x/evmstaking/keeper/keeper_test.go

```diff
@@ -78,16 +77,18 @@ func (s *TestSuite) SetupTest() {
 	cfg.SetBech32PrefixForAccount("story", "storypub")
 	cfg.SetBech32PrefixForValidator("storyvaloper", "storyvaloperpub")
 	cfg.SetBech32PrefixForConsensusNode("storyvalcons", "storyvalconspub")
+	// it should be called after setting the bech32 prefix correctly
+	s.addrs = simtestutil.CreateIncrementalAccounts(4)
```

Comment:
Is this being used after your modification?

Reply:
@limengformal 
`s.addrs` was not originally used anywhere. Since it was added by the story team, I tried to leave it untouched as much as possible, considering there might have been an intended purpose for this code. However, since it’s not being used, should we just remove it?

Reply:
If not used, let's remove it. I think it was originally used to test the removed mocked GetModuleAddress calls.

Reply:
Removed at https://github.com/piplabs/story/pull/61/commits/8112fb03a6fbb6fbd9fe17f06a4523b53f002137